src/lamplib/src/genny/tasks/yaml_linter.py

```diff
@@ -70,8 +73,56 @@ def check_required_fields(yaml_path):
         if field not in workload:
             if field == "Keywords" and yaml_path in GRANDFATHERED_WORKLOADS_WITHOUT_KEYWORDS:
```

Comment:
It's fine if the answer is no, but is it possible we should've done this via the `SchemaVersion` at the top of each YAML file? We're changing required fields in the workload schema, and if that doesn't require a bump, what's the point of even having that field?